### ISSUE TITLE
Using Ziti with Istio

### ISSUE THREAD

**Issue by philipleonardgriffiths** [openziti/ziti-doc#412]
How to deploy Ziti together with Istio, similar to the OSSYS use case where Istio is handling E-W traffic and Ziti handles N-S connectivity for multi-cloud/cluster and remote connectivity for admin & user app access. 

Priority: High.

**Comment by dariuszSki** [comment]
ok, will take look at this unless someone else is already looking at this?

**Comment by qrkourier** [comment]
Yes! We don't have anything written down yet. Here's the basics:

* You need Istio ingresses expressed as Gateway + VirtualService.
* You must configure passthrough TLS for the exposed services.
* you must opt-out of Envoy sidecar injection for the Ziti pods or namespaces.

I'll introduce you to Kevin who's running Istio and Ziti together. If we can capture a few details it'll make documenting this easier.

**Comment by dariuszSki** [comment]
ok, sounds good.
